python/ray/serve/_private/constants.py

```diff
@@ -98,6 +98,7 @@
 DEFAULT_HEALTH_CHECK_PERIOD_S = 10
 DEFAULT_HEALTH_CHECK_TIMEOUT_S = 30
 DEFAULT_MAX_CONCURRENT_QUERIES = 100
+NEW_DEFAULT_MAX_CONCURRENT_QUERIES = 5
```

Comment:
Do you plan to add the migration warning in a separate change?

Reply:
Yeah, I'll make a separate PR for that!